README.md

```diff
@@ -95,6 +95,14 @@ print("Optimized Energy:", atoms.get_potential_energy())
 ```
 
 
+### Finetuning
+You can finetune the model using your custom dataset.
+The dataset should be an ASE sqlite database.
+```python
+python finetune.py --dataset=<dataset_name> --data_path=<your_data_path>
+```
+After the model is finetuned, checkpoints will be saved. You can us the new model and load the checkpoint by modifying the `weights_path` in `pretrained.py`. 
```

Comment:
will, by default, be saved to the `ckpts` folder in the directory you ran the finetuning script from.